### PR TITLE
Apigov 17317

### PR DESCRIPTION
**Pull request by ms8527**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/central/mesh_management/using_redhat_openshift.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The Axway Istio agents are compatible with Red Hat® OpenShift® 4.7+ when using Istio 1.8.2.
+
+## Prerequisites
+
+Before installing and managing an OpenShift cluster to deploy the Istio agents to, you need to have a platform (AWS, Azure, GCP) for installing OpenShift Container Platform (OCP). 
+
+For details to install OpenShift on a selected plaftfrom, please see [OCP guidelines](https://docs.openshift.com/container-platform/4.7/installing/installing-preparing.html)
+
+For example, follow the instructions from [here](https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-default.html#ssh-agent-using_installing-aws-default) to install OpenShift on AWS, start with creating SSH keys and then deploying cluster.
```
Comment (tsjohns9) [inline content/en/docs/central/mesh_management/using_redhat_openshift.md:18]: and then deploying **to a**  cluster.

File: content/en/docs/central/mesh_management/using_redhat_openshift.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Using Istio 1.8
+
+Once the cluster is ready to use, please see [Deploy your agents with the Amplify CLI](/docs/central/mesh_management/deploy-your-agents-with-the-amplify-cli) to learn how to use the CLI to generate Istio and Hybrid agent override files. You will then use these override files to install Istio and your selected agents.
```
Comment (tsjohns9) [inline content/en/docs/central/mesh_management/using_redhat_openshift.md:22]: to learn how to use the CLI to generate ~~Istio and Hybrid agent override files.~~
 learn how to use the CLI to generate **the override files.**

File: content/en/docs/central/mesh_management/using_redhat_openshift.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ 
+## Istio Installation
+Once the Istio override file is generated, merge the generated Istio override file with your Istio configuration. Follow the steps in the [Istio docs](https://istio.io/latest/docs/setup/platform-setup/openshift/) for installing Istio into an OCP cluster. Istio has some set of profiles that we can use for its installation. We can use the appropriate profile and apply that along with istio-override.yaml file
```
Comment (tsjohns9) [inline content/en/docs/central/mesh_management/using_redhat_openshift.md:25]: merge ~~the generated Istio override file~~
merge **it**

Istio has ~~some set of profiles~~ that we can use
Istio has **multiple profiles** that we can use

~~We can use~~ the appropriate profile
**Select** the appropriate profile

File: content/en/docs/central/mesh_management/using_redhat_openshift.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+ ## Istio Traceability Agent installation
+
+The next step is to install the ALS traceability agent that logs and publishes traffic on APICentral. Please follow the instructions [here](/docs/central/mesh_management/traceability_agent_configuration) for setting up ALS Traceability Agent.
```
Comment (tsjohns9) [inline content/en/docs/central/mesh_management/using_redhat_openshift.md:29]: Why is there a section for ALS and not all of the agents?

File: content/en/docs/central/mesh_management/using_redhat_openshift.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+---
+{{< alert title="Public beta" color="warning" >}}This is a preview to create an OCP cluster and install compatible Istio agents.{{< /alert >}}
+
+The Axway Istio agents are compatible with Red Hat® OpenShift® 4.7+ when using Istio 1.8.2.
+
+## Prerequisites
+
+Before installing and managing an OpenShift cluster to deploy the Istio agents to, you need to have a platform (AWS, Azure, GCP) for installing OpenShift Container Platform (OCP). 
+
+For details to install OpenShift on a selected plaftfrom, please see [OCP guidelines](https://docs.openshift.com/container-platform/4.7/installing/installing-preparing.html)
```
Comment (tsjohns9) [inline content/en/docs/central/mesh_management/using_redhat_openshift.md:16]: ~~For details to install OpenShift on a selected plaftfrom~~
**To learn how to install OpenShift on a selected platform**

please see ~~OCP guidelines~~
please see **the OCP documentation for installing a cluster**

File: content/en/docs/central/mesh_management/using_redhat_openshift.md (hunk 4 above)
Comment (ms8527) [reply content/en/docs/central/mesh_management/using_redhat_openshift.md:29]: Hey @markbice 
Do you want me to add another section for installing Discovery agents? Or we need to show the setup for only Traceability

Comment (markbice) [reply content/en/docs/central/mesh_management/using_redhat_openshift.md:29]: we need to handle both here.... but refer to just the Traceability Agent as a separate operation

**Review by markbice — APPROVED** [review]
reviewed and approved

**Pull request merged**